src/base/server_type.cpp

```diff
@@ -261,7 +285,7 @@ passive_tran_server *get_passive_tran_server_ptr ()
     }
   else
     {
-      assert (is_passive_transaction_server ());
+      assert (is_active_transaction_server ());
```

Comment:
I think the assert was ok. It is intended to fail if it is not PTS. It is not intended to return null. In the sense that the function should not be called if the process is not a PTS.
```suggestion
      assert (is_passive_transaction_server ());
```

Reply:
So how about just `assert (0)` being discussed in another thread? : https://github.com/CUBRID/cubrid/pull/3867/files#r994220649